pymtl3/datatypes/BitStruct.py

```diff
@@ -94,6 +81,13 @@ class {name}( BitStruct ):
   def __init__( s, {args_str} ):
 {assign_str}
 
+  def __ilshift__( s, x ):
```

Comment:
@yo96 Can you also implement this recursive ilshift and recursive _flip in the new struct branch?